### PR TITLE
chore(weave): Makes title reflect selected category for table views

### PR DESCRIPTION
**Pull request by tssweeney**
Screenshot 2024-01-25 at 00 27 23
Screenshot 2024-01-25 at 00 27 03
Screenshot 2024-01-25 at 00 26 52

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionsPage.tsx [hunk 1]
```diff
@@ -43,14 +47,32 @@ export const ObjectVersionsPage: React.FC<{
...
+  const {filter, setFilter} = useInitializingFilter(
+    props.initialFilter,
+    props.onFilterUpdate
+  );
+
+  const title = useMemo(() => {
+    if (filter.typeCategory) {
+      return _.capitalize(filter.typeCategory) + ' Objects';
+    }
+    return 'Calls';
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionsPage.tsx:59]: Objects?

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/OpVersionsPage.tsx [hunk 2]
```diff
@@ -46,14 +46,32 @@ export const OpVersionsPage: React.FC<{
...
+  const {filter, setFilter} = useInitializingFilter(
+    props.initialFilter,
+    props.onFilterUpdate
+  );
+
+  const title = useMemo(() => {
+    if (filter.opCategory) {
+      return _.capitalize(filter.opCategory) + ' Operations';
+    }
+    return 'Calls';
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/OpVersionsPage.tsx:58]: Operations?

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/OpVersionsPage.tsx (hunk 2 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/OpVersionsPage.tsx]: 👍

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionsPage.tsx (hunk 1 above)
Comment (tssweeney) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionsPage.tsx]: 👍

**Pull request merged**